tests/checker/test_httpbin.py

```diff
@@ -55,11 +55,7 @@ def test_basic_auth(self):
         password = "testpassword"
         url = get_httpbin_url("/basic-auth/%s/%s" % (user, password))
         nurl = self.norm(url)
-        entry = dict(
-            user=user,
-            password=password,
-            pattern=re.compile(r'.*'),
-        )
+        entry = dict(user=user, password=password, pattern=re.compile(r".*"),)
```

Comment:
```suggestion
        entry = dict(user=user, password=password, pattern=re.compile(r".*"))
```

Reply:
This is a Black bug BTW: https://github.com/psf/black/issues/1202

(it's closed as duplicate, or, rather, one small instance of the larger bug https://github.com/psf/black/issues/1288)